changelog/fix_fix_an_autocorrection_error_for_lint_empty_conditional_body.md

```diff
@@ -0,0 +1 @@
+* [#10951](https://github.com/rubocop/rubocop/issues/10951): Fix an autocorrection error for `Lint/EmptyConditionalBody`. ([@ydah][])
```

Comment:
Can you add a more bit explanation about the context of this incorrect autocorection?